enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/discovery/ClusterMember.java

```diff
@@ -58,4 +59,9 @@
     File storeDir();
 
     File homeDir();
+
+    default void updateConfig( Setting<?> setting, String value )
+    {
+        config().augment( setting, value );
```

Comment:
I think this actually won't work for ReadReplica because it doesn't use the same pattern as CoreClusterMember which uses memberConfig, and instead it uses Config.defaults( config ) in start(). I realise this was already broken and not a fault of your PR.